config/monitoring/tracing/jaeger/105-zipkin-service.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: zipkin
```

Comment:
Should we name this something which doesn't conflict with the Zipkin service (e.g. jaegar-zipkin)? I know running both is likely uncommon but I also don't think there's a need to conflict

Reply:
Using this approach, means it is only the backend that has to change. If a different service name is used, the proxies etc would need to be reconfigured to use a different zipkin address.

Either way, only a single backend can be used - so seems less of an impact if the backend presents the same service address, and we just make sure only one is installed?